examples/stack_llama/scripts/README.md

```diff
@@ -0,0 +1,15 @@
+# RLHF pipeline for the creation of Stack LLaMa" a Stack exchange llama-7b model.
+There were three main steps to the training process:
+1. Supervised fine-tuning of the base llama-7b model to create llama-7b-se:
+    - `python  examples/stack_llama/scripts/sft_stack_exchange_peft.py --model_path=<LLAMA_MODEL_PATH> --streaming --no_gradient_checkpointing --learning_rate 1e-5 --max_steps 5000 --output_dir ./llama-se`
+2. Reward modeling using dialog pairs from the SE dataset using the llama-7b-se to create llama-7b-se-rm:
+    - `deepspeed --num_gpus=8 examples/stack_llama/scripts/reward_modeling_peft.py --model_name=<LLAMA_SE_MODEL> --deepspeed="/fsx/kashif/llama-SE/ds_config.json"`
```

Comment:
We don't really need DS for this right if we use PEFT? can we just use accelerate here? I think this would make things much simpler.

Reply:
cc @kashif

Reply:
Yes I agree, just matching what was used when @kashif trained